src/app/components/playback/playback_control.rs

```diff
@@ -115,18 +115,43 @@ impl PlaybackControl {
         let model = Rc::new(model);
         let debouncer = Debouncer::new();
         let debouncer_clone = debouncer.clone();
-
         let track_position = &widget.track_position;
+
         widget.seek_bar.connect_change_value(
-            clone!(@weak model, @weak track_position => @default-return signal::Inhibit(false), move |_, _, requested| {
+            clone!(@weak model, @weak track_position => @default-return signal::Inhibit(false), move |_, scroll, mut requested| {
+                let mut debounce = 200;
+                match scroll {
+                    gtk::ScrollType::StepForward => {
+                        let duration = model.current_song_duration().unwrap_or(0.0);
+                        requested = if requested + 5000.0 <= duration {
+                            requested + 5000.0
+                        } else { duration };
+                        debounce = 100;
+                    },
+                    gtk::ScrollType::StepBackward => {
+                        requested = if requested >= 5000.0 {
+                            requested - 5000.0
+                        } else { 0.0 };
+                        debounce = 100;
+                    },
+                    _ => (),
+                }
                 track_position.set_text(&format_duration(requested));
-                debouncer_clone.debounce(200, move || {
+                debouncer_clone.debounce(debounce, move || {
                     model.seek_to(requested as u32);
                 });
                 signal::Inhibit(false)
             }),
         );
 
+        widget.seek_bar.connect_scroll_event(move |_, _| {
+            /*
+            Scrolling doesnt work well. Only shows direction if the slider is clicked
```

Comment:
do you mind explaining a bit more, what direction are you referring to? is it actively stopping the rest from working?

Reply:
So when this event is triggered it will send EventScroll with the scroll direction included. Unfortuantley the only way this sends the proper up or down is when the user is clicking the slider knob while scrolling. Otherwise it is always sends as smooth.
The reason I decided to do this instead is to remove an issue where scrolling would not work. (similar to when clicking was not working but that was a setting issue)
Unless it does work and it is once again another setting issue.

Reply:
so when it's sending a ScrollDirection::Smooth, it prevents scrolling to change the value?

because in my case, scrolling offsets the knob in the expected direction so all seems good! it's not incredibly convenient, but it's not broken :thinking:

Reply:
Hmm this might be another configuration issue. For me it does the same thing it did with clicking (before I discoverd the config option).
What are the gtk config options for the flathub builds?